prebackup/prebackup.go

```diff
@@ -61,10 +87,6 @@ func (p *PreBackup) getPodTemplates() (*k8upv1alpha1.PreBackupPodList, error) {
 func (p *PreBackup) generateDeployments(templates *k8upv1alpha1.PreBackupPodList) []appsv1.Deployment {
 	deployments := make([]appsv1.Deployment, 0)
 
-	if len(templates.Items) == 0 {
-		return deployments
-	}
```

Comment:
By removing this, empty list of pre backup pods leads to the condition `ConditionPreBackupPodsReady`. I doubt that's intended?

Reply:
The check has simply [moved up the call-stack](https://github.com/vshn/k8up/pull/204/files/baafb8008fcbcbaab26eae22ca47a0a31c34a777#diff-b539ff69a01fcdd227c6d02d84533cbc0efdfed55761119bd9c125fb21337bd2R64) so that the method is not even called.